dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -958,7 +958,7 @@ def down(preserve_volumes: bool, cleanup_mypy_cache: bool, project_name: str):
 @option_verbose
 @option_dry_run
 @click.argument("exec_args", nargs=-1, type=click.UNPROCESSED)
-def exec(exec_args: tuple):
+def exec_(exec_args: tuple):
```

Comment:
Overall I like the change! But as a nit: I am not a big fan of just adding a `_` as suffix, it feels weird to me. Non blocking though, if others feel it is fine, I am fine

Reply:
Sorry for late response, basically its suggestion from PEP8 or pycodestyle or something similar, I do not really remember when it comes from originally

Particular in this case it is not really matter how to named it because there is interface to cli and not intended to be called directly, I just want to keep original name as close as possible to `exec` which refers to the `breeze exec`

In the another word I do not have any preferences, so if there is any better naming I could easy rename it

Reply:
I think _this_ change specifically, but there are some variables below that could use a better rename than just an underscore suffix.

Reply:
Agree.

Reply:
_There are only two hard things in Computer Science: cache invalidation and naming things_

Reply:
BTW in this place it could be replaced by `exec` -> `execute`

Reply:
> There are only two hard things in Computer Science: cache invalidation and naming things

And off-by-one mistakes